packages/Checkbox/tests/cypress/Checkbox.cypress.js

```diff
@@ -0,0 +1,9 @@
+describe("<CheckBox />", () => {
+  before(() => {});
+
+  it("should be checked when you click the checkbox", () => {
+    cy.visitStorybook("checkbox-automation-tests--cypress");
+    cy.get("label").click();
+    cy.get("label").click();
```

Comment:
This text is actually not expecting any results. If you think we don't need a cypress test, and jest is sufficient then we don't need to add it here.  It seems like this is testing the same thing as jest

Reply:
ok